recipe-server/normandy/recipes/tests/__init__.py

```diff
@@ -108,6 +106,21 @@ def locales(self, create, extracted, **kwargs):
             for locale in extracted:
                 self.locales.add(locale)
 
+    @factory.post_generation
+    def enabled(self, create, extracted, **kwargs):
+        if extracted:
+            approval = ApprovalRequestFactory(revision=self.latest_revision)
+            approval.approve(UserFactory(), 'r+')
```

Comment:
How would a test specify which user is approving a recipe? The implicit creation of a user feels unexpected to me.